docs/resources/add-ins-differences.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: 'Differences between Office Scripts and Office Add-ins'
 description: 'The behavior and API differences between Office Scripts and Office Add-ins.'
-ms.date: 06/01/2020
+ms.date: 06/02/2021
 localization_priority: Normal
 ---
 
 # Differences between Office Scripts and Office Add-ins
 
-Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook a JavaScript API. However, the Office Scripts APIs are a specialized, synchronous version of the Office JavaScript API.
+Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook through JavaScript APIs. However, Office Scripts are designed to be quickly made by anyone looking to improve their workflow. Whereas Office Add-ins are larger-scale integrations for an Office host application.
```

Comment:
Seems odd to follow up a header about "differences" with how much is in common. How about this?
```suggestion
Understand the differences between Office Scripts and Office Add-ins to know when to use each one. Office Scripts are designed to be quickly made by anyone looking to improve their workflow. Office Add-ins integrate with the Office UI for a more interactive experience through ribbon buttons and task panes. Office Add-ins can also expand built-in Excel functions by providing custom functions.
```